tests/cli/convert_mode_test.py

```diff
@@ -142,6 +144,69 @@ def test_pandoc_command_with_correct_args_template(run_mock: Mock):
     )
 
 
+@patch('subprocess.run')
+def test_pandoc_command_with_correct_file_output(run_mock: Mock):
+    """Test that pandoc is called with the -t pdf when --to-pdf
+       is specified.
+    """
+
+    in_file = 'tests/test_documents/ast_error_test_1.md'
+    out_file = 'test/test_documents/out.pdf'
+    pd_command = f'pandoc {in_file} -o {out_file}  --mathjax  -t pdf'  # noqa: E501
+
+    main(['convert', in_file, out_file, '--to-pdf'])
+    run_mock.assert_called_once_with(
+        pd_command,
+        shell=True,
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL,
+    )
+
+
+@patch('subprocess.run')
+def test_pandoc_command_with_correct_file_output_with_template(run_mock: Mock):
+    """Test that pandoc is called with the correct filenames and flags when
+       --to-pdf is passed and also a template is specified.
+    """
+
+    in_file = 'tests/test_documents/ast_error_test_1.md'
+    out_file = 'test/test_documents/out.pdf'
+    pd_template = 'eisvogel.latex'
+    pd_command = f'pandoc {in_file} -o {out_file} --template {pd_template} --mathjax  -t pdf'  # noqa: E501
```

Comment:
What's up with these double spaces? It does not change the functionality but seems weird...